client/transactions/list/index.js

```diff
@@ -149,14 +149,15 @@ export const TransactionsList = ( props ) => {
 		isLoading: isSummaryLoading,
 	} = useTransactionsSummary( getQuery(), props.depositId );
 
-	const columnsArgs = [
```

Comment:
I know this was a neat thing to do, but the "exhaustive deps" rule cannot determine the dependencies when used this way.